EIPS/eip-4672.md

```diff
@@ -0,0 +1,48 @@
+---
+eip: 4672
+title: address.nonce
+description: A new EVM opcode to get the current nonce of an EOA.
+author: 0xfoobar (@0xfoobar)
+discussions-to: https://ethereum-magicians.org/t/eip-4672-account-nonce-opcode/8171
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-01
+requires: none
+---
+
+## Abstract
+
+We propose an additional EVM opcode, `address.nonce` (0x47), which could provide novel protection against sybil attacks and expand the smart contract design space. This would return `0` for an unused externally-owned address (EOA), `0` for a smart contract address, and, for example, `2` for an EOA which has sent 3 transactions. The account.nonce opcode would exactly track the nonce of the last transaction originating from that address.
+
+A full list of EVM opcodes can be tracked here: https://github.com/crytic/evm-opcodes
```

Comment:
I'd recommend removing this external link.

Reply:
done